src/search.js

```diff
@@ -1,23 +1,25 @@
-import {assoc, concat, filter, has, isEmpty, map, pipe, prop, sortBy} from "ramda"
+import {ascend, assoc, concat, descend, has, isEmpty, map, prop, sortBy, sortWith} from "ramda"
 import * as diacritics from 'diacritics'
 
 
 export function findParametersAndVariables(parameters, variables, query) {
-  function matchesQuery(parameterOrVariable) {
-    if (isEmpty(normalizedQuery)) {
-      return true
-    }
-    return parameterOrVariable.name.includes(normalizedQuery) ||
-      parameterOrVariable.searchIndex.includes(normalizedQuery)
+  function indexInString(substring, string) {
+    const index = string.indexOf(substring)
+    return index === -1 ? Number.MAX_SAFE_INTEGER : index
   }
   const normalizedQuery = diacritics.remove(query.toLowerCase())
-  return pipe(
-    filter(matchesQuery),
-    sortBy(prop('name')),
-  )(concat(
+  let parametersAndVariables = concat(
     map(assoc('type', 'parameter'), parameters),
     map(assoc('type', 'variable'), variables),
-  ))
+  )
+  const sortFunction = isEmpty(normalizedQuery)
+    ? sortBy(prop('name'))
+    : sortWith([
+      ascend(item => indexInString(normalizedQuery, item.name)),
+      descend(item => item.searchIndex.includes(normalizedQuery)),
+      ascend(prop('name')),
+    ])
+  return sortFunction(parametersAndVariables)
 }
 
 export function withSearchIndex(propertyName, objects) {
```

Comment:
There is a dumb mistake, sorry. I'm gonna fix it but it should be "description" and not searchIndex! 

The odd part is that it works as if it *was* "description"... I'll fix that very soon.